backend/src/yeomanui.ts

```diff
@@ -73,7 +75,28 @@ export class YeomanUI {
     // on the other hand, we never look for newly installed generators...
 
     const promise: Promise<IPrompt> = new Promise(resolve => {
+      const cwd = path.join(os.homedir(), "projects");
       const env: Environment.Options = Environment.createEnv();
+      const envGetNpmPaths: () => any = env.getNpmPaths;
+      env.getNpmPaths = function (localOnly:any = false) {
+        // Start with the local paths derived by cwd in vscode 
+        // (as opposed to cwd of the plugin host process which is what is used by yeoman/environment)
+        const localPaths: any[] = [];
+    
+        // Walk up the CWD and add `node_modules/` folder lookup on each level
```

Comment:
Need to create a new method called getEnvironment that will return an environment and will contain all the new getNpmPath logic